stripes.js

```diff
@@ -92,8 +93,19 @@ commander
       options: { stripesLoader: stripesLoaderConfig },
     }));
     config.output.path = path.resolve(outputPath);
+    if (publicPath) {
+      config.output.publicPath = publicPath;
+    }
     const compiler = webpack(config);
-    compiler.run(function (err, stats) { });
+    compiler.run(function (err, stats) {
+      // reason: index.html is not generated 
```

Comment:
I'm inclined to make index.html generated rather than place this logic in here.  We can make use of HtmlWebpackPlugin or a custom plugin/template to do the same.  I feel this will be safer for future index modifications.  We can also begin to control bundle name output for cache busting in environments not using publicPath to do so.

Reply:
Totally agree. It would be easier if index.html is generated. Is there a plan to address that soon? Meanwhile is it OK to merge this small localized non-breaking change to meet my immediate need?